osquery/tables/applications/darwin/browser_plugins.cpp

```diff
@@ -90,7 +100,20 @@ QueryData genBrowserPlugins(QueryContext& context) {
     std::vector<std::string> bundles;
     if (listDirectoriesInDirectory(kBrowserPluginsPath, bundles).ok()) {
       for (const auto& dir : bundles) {
-        genBrowserPlugin("0", dir, results);
+        // Check if the 'plugin' we're looking at is the Disabled folder
+        // if it is we enumerate the plugins under that dir, and set a
+        // disabled flag.
+        if(dir.find("Disabled") != std::string::npos){
```

Comment:
There's s bit of code duplication with this conditional (or with the slight refactor mentioned below), perhaps this can be captured in a lambda?